packages/app/src/runner/SpecRunnerHeader.cy.tsx

```diff
@@ -83,6 +85,26 @@ describe('SpecRunnerHeader', () => {
     cy.get('[data-cy="aut-url"]').should('exist')
   })
 
+  it('url section handles long url/small viewport', {
+    viewportWidth: 500,
+  }, () => {
+    const autStore = useAutStore()
+
+    autStore.updateUrl('http://localhost:3000/pretty/long/url.spec.jsx')
+
+    cy.mountFragment(SpecRunnerHeaderFragmentDoc, {
+      onResult: (gql) => {
+        gql.currentTestingType = 'e2e'
+      },
+      render: (gqlVal) => {
+        return renderWithGql(gqlVal)
+      },
+    })
+
+    cy.get('[data-cy="aut-url"]').should('exist')
```

Comment:
Mainly this is to get a Percy snapshot of the state with some more characters in there. I've copied the wrapping behavior of 9.x, but since we have added browser info, and the viewport info is wider, we wrap much earlier than 9.x. There's no new behavior defined atm but I expect that in design review, this will be dialed in some more. But Percy will tell us if we do something that changes it in the meantime.